package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "mattapan-project",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
+    "@types/mapbox-gl": "^2.4.1",
+    "dotenv": "^10.0.0",
+    "eslint-plugin-import": "^2.24.2",
```

Comment:
Should linting be a dev dependency?

Reply:
:+1: good catch, popping it in dev dependency